src/screens/signup-store-wallet.tsx

```diff
@@ -1,32 +1,35 @@
 import React, {useEffect} from 'react';
 
-import {RouteProp, useNavigation, useRoute} from '@react-navigation/native';
-import {StackNavigationProp} from '@react-navigation/stack';
 import {View} from 'react-native';
 
 import {app} from '@app/contexts';
 import {captureException, showModal} from '@app/helpers';
-import {useWallets} from '@app/hooks';
+import {useTypedNavigation, useTypedRoute, useWallets} from '@app/hooks';
+import {I18N, getText} from '@app/i18n';
 import {generateMnemonic} from '@app/services/eth-utils';
-import {RootStackParamList} from '@app/types';
 import {
   ETH_HD_PATH,
   ETH_HD_SHORT_PATH,
   MAIN_ACCOUNT_NAME,
 } from '@app/variables';
 
 export const SignupStoreWalletScreen = () => {
-  const navigation = useNavigation<StackNavigationProp<RootStackParamList>>();
-  const route = useRoute<RouteProp<RootStackParamList, 'createStoreWallet'>>();
+  const navigation = useTypedNavigation();
+  const {nextScreen} = useTypedRoute<'createStoreWallet'>().params;
 
   const wallets = useWallets();
 
   useEffect(() => {
-    showModal('loading', {text: 'Creating an account'});
+    showModal('loading', {
```

Comment:
Create in `@app/helpers` function `showLoadingWithText(text:string)` which will call getText inside

Reply:
done